docs/api/public_php_api.md

```diff
@@ -15,63 +15,63 @@ It will allow you to create, retrieve, update and delete all the eZ Platform obj
 $repository = $container->get( 'ezpublish.api.repository' );
 ```
 
-By itself, the repository doesn't do much. It allows three types of operations: user authentication (getting / changing the current user), issuing transactions, and obtaining services. 
+Repository allows three types of operations: user authentication (getting / changing the current user), issuing transactions, and obtaining services. 
 
 !!! tip "Inline objects documentation"
 
-    Pay attention to the inline phpdoc block in this code stub. It tells your IDE that `$repository` is an instance of  `\eZ\Publish\API\Repository\Repository`. If your IDE supports this feature, you will get code completion on the `$repository` object. This helper is a huge time saver when it comes to learning about the eZ Platform API.
+    Pay attention to the inline `phpdoc` block in this code stub. It tells your IDE that `$repository` is an instance of `\eZ\Publish\API\Repository\Repository`. If your IDE supports this feature, you will get code completion on the `$repository` object. This helper is a huge time saver when it comes to learning about the eZ Platform API.
 
 ### The service container
 
-The above code snippet implies that the [service container](http://symfony.com/doc/2.0/book/service_container.html) is available in the context you are writing your code in.
+The above code snippet implies that the [service container](http://symfony.com/doc/3.4/book/service_container.html) is available in the context you are writing your code in.
```

Comment:
It's 1.x, so we should link to the 2.8 version of Symfony doc (and then change back to 3.4 for v2)
The same below